src/utils/list.zig

```diff
@@ -210,6 +210,10 @@ pub fn List(comptime T: type) type {
 			}
 		}
 
+		pub fn replace(self: *@This(), i: usize, elem: T) void {
+			self.items[i] = elem;
+		}
```

Comment:
This function is unnecessary, zig members are always public, so you can just do `list.items[i] = elem` at the callsite.